### PR TITLE
772 notify fwmt response received

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
For fwmt receipting we may sometimes need to get a case id from the qid.  Where someone has submitted an undressed qid?

# What has changed
New Endpoint added under /cases/qid
Returns an exciting single field dto, caseIdDto

# How to test?
make test
Wait until other work is done and the acceptance tests exercise this properly

# Links
https://trello.com/c/qAx45J4w/772-rmc-190a-notify-fwmt-of-responses-received-8
# Screenshots (if appropriate):

### CODE REVIEW

File: src/main/java/uk/gov/ons/census/caseapisvc/exception/QidNotFoundException.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package uk.gov.ons.census.caseapisvc.exception;
+
+public class QidNotFoundException extends RuntimeException {
+  public QidNotFoundException(String qid) {
+    super(String.format("Qid Not Found '%s' not found", qid));
```
Comment (liamtoozer) [inline src/main/java/uk/gov/ons/census/caseapisvc/exception/QidNotFoundException.java:5]: Picky (sorry!) but remove duplicate 'not found'?

File: src/main/java/uk/gov/ons/census/caseapisvc/exception/UacQidLinkWithNoCaseException.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package uk.gov.ons.census.caseapisvc.exception;
+
+public class UacQidLinkWithNoCaseException extends RuntimeException {
+  public UacQidLinkWithNoCaseException(String qid) {
+    super(String.format("Qid '%s' link but has not case not found", qid));
```
Comment (liamtoozer) [inline src/main/java/uk/gov/ons/census/caseapisvc/exception/UacQidLinkWithNoCaseException.java:5]: Maybe tweak text to something like `"Qid '%s' link found but has no linked case"`?

**Review by ryangrundy7 — APPROVED** [review]
LGTM

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

**Pull request merged**